market_api/market_api.py

```diff
@@ -0,0 +1,29 @@
+import requests
+
+BASE_URL = "http://api.marketcheck.com"
```

Comment:
It's better to keep this kind of constant in a separate file,  like a `settings.py`